UnitTests/Application/MouseTests.cs

```diff
@@ -98,6 +98,11 @@ public void MouseCoordinatesTest_NoFrames (int offset, int clickX, int clickY, i
 		Application.Top.Y = pos.Y;
 		Application.Top.Width = size.Width;
 		Application.Top.Height = size.Height;
+		Assert.Equal (LayoutStyle.Absolute, Application.Top.LayoutStyle);
+		Assert.Equal (new Rect (0, 0, 80, 25), Application.Top.Frame);
+		Application.Top.LayoutStyle = LayoutStyle.Computed;
+		Application.Top.SetRelativeLayout (new Rect (Point.Empty, new Size (Application.Driver.Cols, Application.Driver.Rows)));
+		Assert.Equal (new Rect (offset, offset, 10, 10), Application.Top.Frame);
```

Comment:
With my new change this unit test fails and the right reason. `Application.Top.LayoutStyle` is always `LayoutStyle.Absolute`. Thus the Application.Top.Frame is always the same as `new Rect (Point.Empty, new Size (Application.Driver.Cols, Application.Driver.Rows))`, no matter the `Pos/Dim` are changed. The solution was to change to `LayoutStyle.Computed` and call the `SetRelativeLayout` with the `Driver.Cols/Driver.Rows` dimension.